manifests/monitoring/observability/thanosCompactor.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
```

Comment:
`s/thanosCompactor.yaml/thanos-compactor.yml/` as it is more consistent with existing naming conventions

Reply:
Sure